icons/squirrel.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16 22v-2.3a2.7 2.7 0 0 1 2.7-2.7H20a2 2 0 0 0 2-2v-1c0-2.8-2.2-5-5-5V5a4 4 0 0 0-4 4v2a7 7 0 0 0-7 7V6a2 2 0 0 0 4 0 4 4 0 0 0-8 0v12a4 4 0 0 0 4 4h12" />
+  <path d="M15.6 22a3.5 3.5 0 0 0-3.2-5" />
+  <path d="M18 13h.01" />
```

Comment:
The eye needs to be moved to have 2px space in all directions.
```suggestion
  <path d="M16 22v-2.3a2.7 2.7 0 0 1 2.7-2.7H20a2 2 0 0 0 2-2v-1a5 5 0 0 0-5-5V5a4 4 0 0 0-4 4v2a7 7 0 0 0-7 7V6a2 2 0 0 0 4 0 4 4 0 0 0-8 0v12a4 4 0 0 0 4 4h12" />
  <path d="M15.6 22a3.5 3.5 0 0 0-3.2-5" />
  <path d="M 17.5 13h0.01" />
```
![icon (44)](https://user-images.githubusercontent.com/25524993/236695544-93af773f-2b41-436d-b46e-184e9a557729.svg)